From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Fri, 26 May 2023 11:41:19 +0200
Subject: [PATCH] HACK: don't build tests

The tests can introduce additional dependencies (e.g. xcb-cursor for xwayland)
that are not otherwise needed. The tests are not used in ptxdist anyways, so
just don't build them.

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 tests/meson.build | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/tests/meson.build b/tests/meson.build
index 1d59a93e121c..b7c4412bfdf8 100644
--- a/tests/meson.build
+++ b/tests/meson.build
@@ -1,3 +1,5 @@
+subdir_done()
+
 plugin_test_shell_desktop = shared_library(
 	'weston-test-desktop-shell',
 	'weston-test-desktop-shell.c',
